dlp/snippets/Templates/templates_test.py

```diff
@@ -15,11 +15,15 @@
 import os
 import uuid
 
+import create_inspect_template as ct
+import delete_inspect_template as dt
+
 import google.api_core.exceptions
 import google.cloud.storage
-import pytest
 
-import templates
+import list_inspect_templates as lt
```

Comment:
nit: can you group imports from line 18,19 and 24 together?

Reply:
They are placed that way intentionally to avoid linting errors.